ansible/inventory.yml

```diff
@@ -130,6 +130,9 @@ hosts:
         ubuntu1604-arm64_odroid_c2-2: {ip: 70.167.220.148, user: odroid}
         ubuntu1604-arm64_odroid_c2-3: {ip: 70.167.220.149, user: odroid}
 
+    - macstadium:
+        macos10.12-x64-1: {ip: 207.254.58.162, port: 5, user: administrator}
+
```

Comment:
Is that the right port ? I thought it was going to be something like 10005

Reply:
This is just a test machine that I spun up, ignore the port for now